README.md

```diff
@@ -17,6 +17,34 @@ The templates in the [openshift/templates](./openshift/templates) will build and
 
 The Helm chart deploys a standard StatefulSet. This deployment should work on [OpenShift Local](https://github.com/crc-org/crc), [kind](https://kind.sigs.k8s.io/) or even [Docker Desktop](https://docs.docker.com/desktop/kubernetes/)
 
+## OpenShift or OpenShift Local
+
+1. Optiona: create a namespace
+```
+oc new-project clamav-demo
+
+```
+2. Import the build config template and create the build config
+
+```
+oc apply -f openshift/clamav-bc.conf -n clamav-demo
+oc new-app --template=clamav
+```
+
+3. Start a build
+```
+oc start-build clamav-build
+```
+
+4. When the build is complete, new image stream tag should be avilable, `clamav` with a single image stream tag `clamav:latest`
+
+5. Deploy, if running locally be sure to set the clamav.freshclam.mirrors
+
+```
+cd charts
+helm install clamav clamav --set clamav.image=image-registry.openshift-image-registry.svc:5000/clamav-demo/clamav --set clamav.freshclam.mirrors=https://db.us.clamav.net
+```
```

Comment:
This should not be the default as the entirety of openshift will get ratelimited (use something like https://clamav-mirror-6cdc9e-dev.apps.silver.devops.gov.bc.ca)  Reaching outside openshift to update clamav will ban us from hitting it too often.

Reply:
This part of the readme says if you are running locally, ie OpenShift local.

As for the mirrors, they are defaulted to the bcgov mirror if the user does not override. In the configmap,

https://github.com/pbolduc/bcgov-clamav/blob/e542aefe26be203d025d69cb61c5f7cac89b4e7a/charts/clamav/templates/configmap.yaml#L691-L692

and in the default values file,

https://github.com/pbolduc/bcgov-clamav/blob/e542aefe26be203d025d69cb61c5f7cac89b4e7a/charts/clamav/values.yaml#L24-L26

Reply:
The default mirror is `https://clamav-mirror.apps.silver.devops.gov.bc.ca`